charts/seed-monitoring/charts/core/charts/prometheus/templates/prometheus-vpa.yaml

```diff
@@ -10,3 +10,16 @@ spec:
     name: prometheus
   updatePolicy:
     updateMode: "Auto"
+  resourcePolicy:
+    containerPolicies:
+    # Due to CVE-2019-5736 runC is initially loaded into memory when the container starts.
+    # After some time VPA recommends less memory (2,3Mb) than the size of of runC binary (about 10Mb).
+    # This results in an error when trying to start the container:
+    # failed to write 2485760 to memory.limit_in_bytes in /sys/fs/cgroup/memory/kubepods/prometheus-config-reloader/memory.limit_in_bytes: device or resource busy
+    # https://github.com/lxc/lxc/commit/6400238d08cdf1ca20d49bafb85f4e224348bf9d
+    # https://github.com/helm/charts/issues/11447#issuecomment-464716379
+    - containerName: prometheus-config-reloader
+      mode: "Off"
+      minAllowed:
```

Comment:
Does minAllow considered when the mode is `Off`?

Reply:
I haven't look at  the VPA admission code to be 100% sure. Probably not. But it's always better to be sure that it'll take those values.

Reply:
> Probably not.

Ok, then let's remove it.

Reply:
There was some sync time before the recommendation was removed for `Off` containers.  So it appears that `minAllowed` is not needed in this case